homeassistant/components/fritzbox_callmonitor/sensor.py

```diff
@@ -133,21 +156,48 @@ def name(self):
         """Return the name of the sensor."""
         return self._name
 
+    @property
+    def icon(self):
+        """Return the icon of the sensor."""
+        return ICON_PHONE
+
     @property
     def device_state_attributes(self):
         """Return the state attributes."""
+        if self._prefixes:
+            self._attributes[ATTR_PREFIXES] = self._prefixes
+        if self._phonebook_name:
+            self._attributes[ATTR_PHONEBOOK_NAME] = self._phonebook_name
+        if self._phonebook_url:
+            self._attributes[ATTR_PHONEBOOK_URL] = self._phonebook_url
         return self._attributes
 
+    @property
+    def device_info(self):
+        """Return device specific attributes."""
+        return {
+            "name": self._phonebook.fph.modelname,
+            "identifiers": {(DOMAIN, self._host, self._phonebook_id)},
+            "manufacturer": MANUFACTURER,
+            "model": self._phonebook.fph.modelname,
+            "sw_version": self._phonebook.fph.fc.system_version,
+        }
+
+    @property
+    def unique_id(self):
+        """Return the unique ID of the device."""
+        return f"{self._host}-{self._phonebook_id}"
```

Comment:
We shouldn't use host. That could be different ?

Do we have access to a serial number, mac address or otherwise another unique identifier that doesn't change?